pkg/backends/ghz/backend.go

```diff
@@ -168,16 +169,23 @@ func (b *Backend) Sync(ctx context.Context, loadTest loadTestV1.LoadTest, report
 		mounts  = make([]coreV1.VolumeMount, 1)
 	)
 
-	volumes[0], mounts[0] = NewFileVolumeAndMount("testfile", tfCfgMap.Name, configFileName)
+	volumes[0], mounts[0] = NewFileVolumeAndMount("testfile", tfCfgMap.Name, configFile)
 
 	if tdCfgMap != nil {
 		v, m := NewFileVolumeAndMount("testdata", tdCfgMap.Name, testdataFileName)
 		volumes = append(volumes, v)
 		mounts = append(mounts, m)
 	}
 
+	// Build job args
+	args, err := NewCommandArgs(configFile)
+	if err != nil {
+		b.logger.Error("Error creating job args", zap.String("configFile", configFile), zap.Error(err))
+		return err
+	}
+
```

Comment:
Since this is only used in `NewJob()` and is not a kube resource like the things above; what do you think about moving this as a private function in `resources.go` and calling that in `NewJob()` directly?

We can guess the config extension in it as well.

Reply:
> We can guess the config extension in it as well.

As it's being referenced in both configmap and volume creation, I kept it at the `backend.go`, do you see a better way @s4nji ?

Reply:
Looks like we can just drop the extension for the file name: it seems to [try to parse][1] it in toml and json if it has no matching extension 🤔 

[1]: https://github.com/jinzhu/configor/blob/591dd1903f49b827983691a64ec1e40ff53d05bc/utils.go#L101

Reply:
We can leave a comment somewhere noting that ghz will try to guess the config file format for posterity